code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -389,12 +389,14 @@
 	color = "#FFEBEB"
 
 /datum/reagent/medicine/synthflesh/reaction_mob(mob/living/M, method=TOUCH, reac_volume,show_message = 1)
+	var/healmod = 1
 	if(iscarbon(M))
 		if (M.stat == DEAD)
+			healmod = 2
 			show_message = 0
 		if(method in list(PATCH, TOUCH))
-			M.adjustBruteLoss(-1.25 * reac_volume)
-			M.adjustFireLoss(-1.25 * reac_volume)
+			M.adjustBruteLoss(-0.75 * reac_volume * healmod)
```

Comment:
0.625
1.25/2 = 0.625

Reply:
![image](https://user-images.githubusercontent.com/24857008/85371013-9e131e00-b4fd-11ea-8130-34a0c4b02ce4.png)

Reply:
tweak: synthflesh now heals at half efficiency when the target is still alive